cloud/aws/fargate/ecs.tf

```diff
@@ -23,6 +23,54 @@ resource "aws_security_group" "ecs_tasks" {
   }
 }
 
+resource "aws_security_group" "efs" {
+  name        = "${module.env.module_name}_${var.name}_efs_${module.env.stage}"
+  description = "allow inbound nfs access"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    protocol        = "tcp"
+    from_port       = 2049
+    to_port         = 2049
+    security_groups = [aws_security_group.ecs_tasks.id]
+  }
+
+}
+
+resource "aws_efs_file_system" "efs_volume" {
+  count = var.storage_type == "EFS" ? 1 : 0
```

Comment:
Is it possible to create a module that contains all of the EFS-related logic? I see this ternary expression popping up several times, and moving the logic into a module might allow us to only make the EFS yes / no decision only once.

Reply:
Good idea! I am used to do only one level of modules but here it would be very appropriate to have a second one 😄. Thanks!